emm/indexing/pandas_normalized_tfidf.py

```diff
@@ -37,8 +37,6 @@
 class PandasNormalizedTfidfVectorizer(TfidfVectorizer):
     """Implementation of customized TFIDF vectorizer"""
 
-    dtype = np.float32
```

Comment:
If we remove this, don't we need to set the default value (float32) elsewhere?

Reply:
What's the argument for removing this, shouldn't we keep this in?

Reply:
Was thinking that it's connected to `PandasCosSimIndexer` which passes it's default `np.float32` `dtype` to it. Have put it back as default value in `kwargs` for use cases with different setups.